lib/fx/adapters/postgres/functions.rb

```diff
@@ -11,13 +11,16 @@ class Functions
         FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-EOS.freeze
           SELECT
               pp.proname AS name,
+              pg_get_function_identity_arguments(pp.oid) AS arguments,
               pg_get_functiondef(pp.oid) AS definition
           FROM pg_proc pp
           JOIN pg_namespace pn
               ON pn.oid = pp.pronamespace
           LEFT JOIN pg_depend pd
               ON pd.objid = pp.oid AND pd.deptype = 'e'
-          WHERE pn.nspname = 'public' AND pd.objid IS NULL
+          WHERE pn.nspname = 'public'
```

Comment:
Since this is covered by #11 - what do you think of excluding this in this PR as it is unrelated to this feature?